src/integrationTest/sam.test.ts

```diff
@@ -263,6 +264,13 @@ describe('SAM Integration Tests', async function() {
 
                 it('invokes and attaches on debug request (F5)', async function() {
                     setTestTimeout(this.test?.fullTitle(), 60000)
+                    // Allow previous sessions to go away.
+                    waitUntil(
```

Comment:
Does this need to be `await`ed? Also, should this handle an `undefined` return (if I'm reading this correctly, if the timeout is exceeded and activeDebugSession is still defined, this won't halt the test)?

Reply:
ahhhhhhhh thank you

Reply:
> Also, should this handle an `undefined` return (if I'm reading this correctly, if the timeout is exceeded and activeDebugSession is still defined, this won't halt the test)?

The next line checks the value.